examples/spark2/src/main/java/org/apache/carbondata/examples/sdk/SDKS3Example.java

```diff
@@ -44,34 +44,14 @@ public static void main(String[] args) throws Exception {
             num = Integer.parseInt(args[4]);
         }
 
-        Boolean persistSchema = true;
-        if (args.length > 5) {
-            if (args[5].equalsIgnoreCase("true")) {
-                persistSchema = true;
-            } else {
-                persistSchema = false;
-            }
-        }
-
-        Boolean transactionalTable = true;
-        if (args.length > 6) {
-            if (args[6].equalsIgnoreCase("true")) {
-                transactionalTable = true;
-            } else {
-                transactionalTable = false;
-            }
-        }
-
         Field[] fields = new Field[2];
         fields[0] = new Field("name", DataTypes.STRING);
         fields[1] = new Field("age", DataTypes.INT);
         CarbonWriterBuilder builder = CarbonWriter.builder()
                 .setAccessKey(args[0])
                 .setSecretKey(args[1])
                 .setEndPoint(args[2])
-                .outputPath(path)
-                .persistSchemaFile(persistSchema)
-                .isTransactionalTable(transactionalTable);
```

Comment:
remove this one , this example can't change  value of isTransactionalTable and persistSchemaFile when we test it.  I suggest it better to keep it.